app/src/docs/content/products/dataUnions.mdx

```diff
@@ -1,3 +1,6 @@
+import { Link } from 'react-router-dom'
+import links from '$shared/../links'
+
 # Data Unions
 
-***Documentation for Data Unions is coming soon.***
+Learn more about <Link to={links.docs.dataUnions.root}>Data unions</Link>.
```

Comment:
This needs to be expanded

Reply:
Depends on the final structure for all this, but there is a precedent for duplicate content in favour of placeholder redirects, if it comes to it, e.g. 

`<Route exact path={docs.streams.usingStreamsViaApi} component={UsingStreamsViaApiDocsPage} key="usingStreamsViaApi" />`

`<Route exact path={docs.api.usingStreamsViaApi} component={UsingStreamsViaApiDocsPage} key="usingStreamsViaApi" />,`

The same content file `usingStreamsViaApi.mdx` (Streams via API) is used in the **Streams** and **API** sections.